cou/steps/hypervisor.py

```diff
@@ -159,6 +159,13 @@ def azs(self) -> AZs:
                     logger.debug("skipping machine %s", unit.machine.machine_id)
                     continue
 
+                unit_os_release = app.unit_max_os_version(unit)
+                if unit_os_release >= target:
+                    logger.debug(
```

Comment:
I think we should use info here, instead of debug. It's really important information.